PageObjectPattern/src/test/java/apiSerenityStep/SpotifyApiStep.java

```diff
@@ -0,0 +1,114 @@
+package apiSerenityStep;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+import static org.hamcrest.Matchers.*;
+
+public class SpotifyApiStep {
+    String baseUrl = "https://api.spotify.com";
+    String baseAppTokenUrl = "https://accounts.spotify.com/api/token";
+    String userEmail = "yulia.borisowa.qa@gmail.com";
+    String accessToken = "BQCoROD3yXhukxLeIlNj0iP8zve6-Lbbc_t19WBkDecjHosz3WwT1mEfCPn5WfL5xHC3QoqKQx5_vcVXwVgNJJJqwrTD2Bm-VbPH3CSrWesZKRzz4PyawpCrMd8jXx44Hh1Vz0zgQ-_IMHLFmTuS9RaqNLDWDYlxCLJV5uRZWtZ52NVmizokBOJos4aO9ZJGHRIYsUwOnWtygMPVKFhe54_ccIXp9y08pnf9Vx7mLA92uozxTphUpLNPGYo-E-9NG2fdsbuLlOveb2SMhWLO";
+    String client_id = "07110c2def0d410d8a7894b92b6deca0";
+    String client_secret = "cb71cb414028401086f78986e943636e";
+    String appToken;
+    Response responseGet;
+    Response responsePost;
+    String playListName;
+
+    // useless step for now to implement later TO DO
+    @Step("authorization, token generated manually on website")
```

Comment:
Here you can also use the same split as with Step Definitions classes, i.e. you can create a BaseSteps.java class that will contain all common fields/methods and then create a separate Steps.java class for each particular endpoint that you need.